nrfx/hal/nrf_nvmc.h

```diff
@@ -282,6 +322,24 @@ NRF_STATIC_INLINE void nrf_nvmc_nonsecure_mode_set(NRF_NVMC_Type *    p_reg,
 }
 #endif
 
+NRF_STATIC_INLINE void nrf_nvmc_word_write(uint32_t address,
+                                           uint32_t value)
+{
+    *(volatile uint32_t *)address = value;
+}
+
+NRF_STATIC_INLINE uint32_t nrf_nvmc_word_read(uint32_t address)
+{
+    return *(volatile uint32_t *)address;
+}
+
+NRF_STATIC_INLINE void nrf_nvmc_buffer_read(void *   dst,
+                                            uint32_t address,
+                                            size_t   num_words)
+{
+    memcpy(dst, (void *)address, num_words);
```

Comment:
`memcpy()` operates on bytes, not words.

To avoid confusion and for similarity with:
https://github.com/zephyrproject-rtos/hal_nordic/blob/140140ea047f441fe076d26f79eb54dc9a38bcb6/nrfx/drivers/include/nrfx_nvmc.h#L235
I'd suggest to define this function as:
```
NRF_STATIC_INLINE void nrf_nvmc_bytes_read(void *   dst,
                                           uint32_t address,
                                           uint32_t num_bytes)
```

Reply:
Correct indeed, I applied to quick the requests from the previous review